inspirehep/config.py

```diff
@@ -287,7 +287,7 @@ def _(x):
     "doc_type": ["facet_inspire_doc_type"],
     "formulas": ["facet_formulas"],
     "affiliation": ["authors.affiliations.value", "corporate_author"],
-    "reportnumber": ["report_numbers.value", "arxiv_eprints.value"],
+    "parent_report_number": ["report_numbers.value", "arxiv_eprints.value"],
```

Comment:
This substitution is not correct. This should stay indeed `reportnumber` (it's part of the search syntax). I will amend it myself when cherry picking.